websites/S/signed.host/presence.ts

```diff
@@ -0,0 +1,121 @@
+const presence = new Presence({
+	clientId: "1000664778404016230",
+});
+
+let lastUnique: string | undefined,
+	lastTimestamp: number | undefined = Date.now();
+
+presence.on("UpdateData", async () => {
+	if (document.querySelector(".next-error-h1")) return presence.clearActivity();
+
+	const { pathname, hostname } = document.location;
+
+	if (lastUnique !== pathname) {
+		lastUnique = pathname;
+		lastTimestamp = Date.now();
+	}
+
+	let details = `Currently on ${pathname}`,
+		buttonLabel = "Visit page",
+		buttonURL = document.location.toString(),
+		state = null;
+
+	const loggedUsername = document.querySelector("#premid-username")?.innerHTML;
+
+	switch (hostname) {
+		case "signed.host":
+			if (pathname.includes("/dash")) {
+				loggedUsername
+					? (state = `Logged in as ${loggedUsername}`)
+					: (state = "Not logged in!");
+				buttonLabel = "Join signed.host";
+				buttonURL = "https://signed.host/";
```

Comment:
Buttons that lead to main pages are [prohibited](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets).

```suggestion
```